Description: link aginst gobject for symbols used by telepathy-qt?-farstream
Author: Diane Trout <diane@ghic.org>

--- a/TelepathyQt/Farstream/CMakeLists.txt
+++ b/TelepathyQt/Farstream/CMakeLists.txt
@@ -43,6 +43,7 @@
         ${QT_QTCORE_LIBRARY}
         ${TELEPATHY_FARSTREAM_LIBRARIES}
         ${TELEPATHY_GLIB_LIBRARIES}
+        ${GOBJECT_LIBRARIES}
         telepathy-qt${QT_VERSION_MAJOR}
         ${TP_QT_LIBRARY_LINKER_FLAGS})
 
